tools/imx8mimage.c

```diff
@@ -455,6 +514,14 @@ void build_image(int ofd)
 
 	header_image_off = file_off + ivt_offset;
 
+#ifdef CONFIG_FSPI_CONF_HEADER
+
+	fspi_fd = open("fspi_header.bin", O_RDWR | O_CREAT, S_IRWXU);
```

Comment:
NAK, hardcoded filename? Should be config parameter, so that other board users would have an option to re-define it.